src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -585,12 +584,20 @@ public virtual Loadout MergeLoadouts(Loadout loadoutA, Loadout loadoutB)
     /// <returns></returns>
     public virtual async Task<DiskStateTree> Apply(Loadout loadout, bool forceSkipIngest = false)
     {
+        // Note(Sewer) If the last loadout was a marker loadout, we need to guard
+        // in the case the user has modified the game folder since.
```

Comment:
Reword this a bit, 
If the last loadout was a marker loadout, we we need to skip ingest and ignore changes, since marker loadouts should not be changed.